docs/html/user_guide.rst

```diff
@@ -49,6 +49,22 @@ For more information and examples, see the :ref:`pip install` reference.
 .. _PyPI: https://pypi.org/
 
 
+Basic Authentication Credentials
+********************************
+
+pip supports basic authentication credentials. Basically, in the url there is
+a username and password separated by ``:``.
+
+  ::
+
+    https://[username[:password]]@pypi.company.com/simple
+    0123456789abcdef@pypi.company.com
+    https://aniruddha%24basak:gdg%24js%5Ejf%26l@pypi.company.com
+
+`Here <https://en.wikipedia.org/wiki/Percent-encoding>`_ you can find more about
```

Comment:
I would link directly to the part of the article that has the table [here](https://en.wikipedia.org/wiki/Percent-encoding#Percent-encoding_reserved_characters) and move this link into the paragraph I suggested above.

Reply:
@chrahunt should I add the [table](https://en.wikipedia.org/wiki/Percent-encoding#Percent-encoding_reserved_characters) at the last of the [user_guide.rst](https://github.com/aniruddha2000/pip/blob/basic_cred/docs/html/user_guide.rst) file and `here` will internally link to that table?

Reply:
I don't think it is necessary. By the time the Wikipedia article changes we may have already moved to a different scheme for providing user credentials. :)

Reply:
ok @chrahunt :)

Reply:
I would delete these lines now and insert the link I commented above.